### ISSUE TITLE
Consistently Using TestIDs and Accessibility labels

### ISSUE THREAD

**Issue by cvarjao** [bcgov/bc-wallet-mobile#1514]
Any user interactable/pressable element should have TestID or Accessibility Label

Ideas:
- Add to PR checklist
- Static code analisys / Linting
